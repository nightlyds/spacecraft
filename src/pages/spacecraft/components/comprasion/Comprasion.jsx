import React from "react";
import Translate from "react-translate-component";

const Comprasion = () => {
  return (
    <div className="comprasion-labels">
      <div className="comprasion-labels-rocket-name">
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="rocketName"
            className="comprasion-labels-label"
          />
        </div>
      </div>
      <div className="comprasion-labels-details">
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="firstFlight"
            className="comprasion-labels-label"
          />
        </div>
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="country"
            className="comprasion-labels-label"
          />
        </div>
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="successRate"
            className="comprasion-labels-label"
          />
        </div>
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="costs"
            className="comprasion-labels-label"
          />
        </div>
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="engines"
            className="comprasion-labels-label"
          />
        </div>
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="enginesVersion"
            className="comprasion-labels-label"
          />
        </div>
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="height"
            className="comprasion-labels-label"
          />
        </div>
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="diameter"
            className="comprasion-labels-label"
          />
        </div>
        <div className="comprasion-labels-label-box">
          <Translate
            component="span"
            content="mass"
            className="comprasion-labels-label"
          />
        </div>
      </div>
    </div>
  );
};

export default Comprasion;
