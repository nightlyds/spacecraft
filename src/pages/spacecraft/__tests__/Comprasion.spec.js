import React from 'react';
import Comprasion from '../components/comprasion/Comprasion.jsx';
import { shallow } from 'enzyme';

const setUp = (props) => shallow(<Comprasion {...props} />)

describe("Comprasion component test", () => {
    let component;

    beforeEach(() => {
        component = setUp();
    })

    it("Snapshot test", () => {
        expect(component).toMatchSnapshot();
    })
})