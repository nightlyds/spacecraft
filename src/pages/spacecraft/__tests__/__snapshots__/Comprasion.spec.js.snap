// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comprasion component test Snapshot test 1`] = `
<div
  className="comprasion-labels"
>
  <div
    className="comprasion-labels-rocket-name"
  >
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="rocketName"
      />
    </div>
  </div>
  <div
    className="comprasion-labels-details"
  >
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="firstFlight"
      />
    </div>
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="country"
      />
    </div>
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="successRate"
      />
    </div>
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="costs"
      />
    </div>
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="engines"
      />
    </div>
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="enginesVersion"
      />
    </div>
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="height"
      />
    </div>
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="diameter"
      />
    </div>
    <div
      className="comprasion-labels-label-box"
    >
      <Translate
        className="comprasion-labels-label"
        component="span"
        content="mass"
      />
    </div>
  </div>
</div>
`;
